hoot-core/src/main/cpp/hoot/core/io/OgrUtilities.h

```diff
@@ -28,36 +28,57 @@
 #define OGRUTILITIES_H
 
 // GDAL
-class OGRDataSource;
+#include <gdal.h>
+// Forward declaration
+class GDALDataset;
 
 // Qt
 #include <QString>
 
 // Tgs
 #include <tgs/SharedPtr.h>
 
+#include <vector>
+
 namespace hoot
 {
 
+class OgrDriverInfo
+{
+public:
+  OgrDriverInfo(const char* indicator = NULL, const char* driverName = NULL, bool is_ext = false, unsigned int driverType = GDAL_OF_ALL)
```

Comment:
I think this constructor deserves some comments explaining the parameters. Like is "indicator" a file extension?